app/mailers/approval_notifier_mailer.rb

```diff
@@ -2,14 +2,18 @@ class ApprovalNotifierMailer < ApplicationMailer
   attr_reader :user
 
   def registration_approval_email(user_id:)
-    @user = User.find(user_id)
-    subject = I18n.t("registration_approval_email.subject",
-      full_name: @user.full_name,
-      org_name: @user.facility_group.organization.name)
-    mail(subject: subject,
-         to: supervisor_emails,
-         cc: organization_owner_emails,
-         bcc: owner_emails)
+    if ENV["SIMPLE_SERVER_ENV"] == "production"
```

Comment:
I think you can simply use `SIMPLE_SERVER_ENV` directly since that constant is set [here](https://github.com/simpledotorg/simple-server/blob/master/config/application.rb#L22).